internal/helm/helmchart_structure_exists.go

```diff
@@ -53,8 +54,8 @@ func (r *HelmStructureExists) Test() *txqualitychecks.QualityResult {
 		"LICENSE",
 		"README.md",
 		"values.yaml",
-		"templates",
-		"templates/NOTES.txt",
+		// "templates",
```

Comment:
Prefer deleting code instead of commenting out. We have a version history in case someone needs to know